JEasyCryptoConsole/src/EasyCryptoConsole/EasyCryptoConsole.java

```diff
@@ -6,58 +6,59 @@
 
 import easycrypto.EasyCryptoAPI;
 
+public class EasyCryptoConsole {
 
+    public static void main(String[] args) {
+        boolean output = false;
 
-public class EasyCryptoConsole {
+        try {
+            System.setOut(new PrintStream(System.out, true, "UTF-8"));
+            System.setProperty("file.encoding", "UTF-8");
+            Console console = System.console();
+
+            console.printf("Welcome to CryptoClient!\n");
+            console.printf("Supported methods are: %s\n", EasyCryptoAPI.methods());
+
+            while (true) {
```

Comment:
We have two nested `while` loops here. In general one really should avoid that. Can you come up with a fix?

Reply:
This is definitely a problem we'll fix. Thanks.